releases/v1.2.12.toml

```diff
@@ -0,0 +1,38 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.11"
+
+pre_release = false
+
+preface = """\
+The twelfth patch release for `containerd` 1.2 includes an updated runc with
+a fix for CVE-2019-19921, an updated version of the opencontainers/selinux
+dependency, which includes a fix for CVE-2019-16884, an updated version of the
+`gopkg.in/yaml.v2` dependency to address CVE-2019-11253, and a Golang update.
+
+### Notable Updates
+
+* Update the runc vendor to v1.0.0-rc10 which includes a mitigation for [CVE-2019-19921](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-19921).
+* Update the opencontainers/selinux which includes a mitigation for [CVE-2019-16884](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-16884).
```

Comment:
Not sure if this one should be mentioned; IIUC the CVE was in the runc _binary_, and the updated dependency has no effect in this code-base (please double check); if so, I'm inclined to remove the mention here and above